client/lcd/swagger-ui/swagger.yaml

```diff
@@ -251,7 +251,7 @@ paths:
         - in: query
           name: message.action
           type: string
-          description: "transaction events such as 'message.action=send' which results in the following endpoint: 'GET /txs?message.action=send'"
+          description: "transaction events such as 'message.action=send' which results in the following endpoint: 'GET /txs?message.action=send'. note that each module documents its own events. look for xx_events.md in the corresponding cosmos-sdk/docs/spec directory"
```

Comment:
Not too familiar with Swagger yaml specs, but can we break these strings up onto multiple lines to make it easier to read?

Reply:
This will render properly. Lets not block merge here on this?